sources/api/schnauzer/README.md

```diff
@@ -34,29 +37,78 @@ std = { version = "v1", helpers = ["base64_decode"] } # ... or use the object fo
 ```
 
 ### The schnauzer Library
-The primary user interface is provided via `schnauzer::render_template` and `schnauzer::render_template_file`.
-These functions require the user to pass the template, as well as a `TemplateImporter`, which tells schnauzer how
-to resolve references to settings extensions and the requested helpers.
+The primary user interface is provided via `schnauzer::render_template` and
+`schnauzer::render_template_file`. These functions require the user to pass the template, as
+well as a `TemplateImporter`, which tells schnauzer how to resolve references to settings
+extensions and the requested helpers.
 
-Most users will want to use `schnauzer::BottlerocketTemplateImporter`, which uses settings extensions to resolve
-settings and helper data; however, custom `TemplateImporter` implementations can be used as well.
+Most users will want to use `schnauzer::BottlerocketTemplateImporter`, which uses settings
+extensions to resolve settings and helper data; however, custom `TemplateImporter`
+implementations can be used as well.
 
 For static datasets to be used for tests, enable the `testfakes` feature in `Cargo.toml`.
 
 ### The schnauzer Settings Generator
-Bottlerocket settings can be generated via `schnauzer-v2` (pending rename to `schnauzer`), which emits rendered
-templates as JSON strings compatible with the Bottlerocket API.
-The setting generator allows the user to specify any template requirements via the CLI, rather than requiring
-frontmatter, e.g.
-`schnauzer-v2 render --requires 'settings@v1(helpers=[myhelper])' --template 'foo-{{ myhelper settings.bar }}'`.
+Bottlerocket settings can be generated via `schnauzer-v2` (pending rename to `schnauzer`), which
+emits rendered templates as JSON strings compatible with the Bottlerocket API.
+The setting generator allows the user to specify any template requirements via the CLI, rather
+than requiring frontmatter, e.g.
```

Comment:
Is "rather than requiring frontmatter" something that should also be mentioned where frontmatter is introduced above?

Reply:
Good idea, I'll add this in a separate commit.